common/beerocks/bwl/dummy/base_wlan_hal_dummy.cpp

```diff
@@ -83,8 +83,20 @@ base_wlan_hal_dummy::base_wlan_hal_dummy(HALType type, std::string iface_name, b
     }
 
     // Set up dummy external events fd
-    if ((m_fd_ext_events = eventfd(0, EFD_SEMAPHORE)) < 0) {
-        LOG(FATAL) << "Failed creating eventfd: " << strerror(errno);
+    // dummy implementation is based on monitoring text file changes which
+    // are used to simulate clients activity - STA connected, disconnected, etc.
+    // For that we are creating the INOTIFY instance
+    // inotify_init1 not available with older kernels, consequently inotify reads block.
+    // inotify_init1 allows directory events to complete immediately, avoiding buffering delays. In practice,
+    // this significantly improves monotiring of newly created subdirectories.
+#ifdef IN_NONBLOCK
```

Comment:
And if you explained to use one instead of the other according to the kernel version, then why didn't you do it?
Edit: Ok I think I got it! The `IN_NONBLOCK` is not defined in an old kernel version. I think it is worth mention.

Reply:
ok